packages/kbn-router-to-openapispec/src/process_router.ts

```diff
@@ -63,10 +64,19 @@ export const processRouter = (
         parameters.push(...pathObjects, ...queryObjects);
       }
 
+      let description = '';
+      if (route.security) {
+        const authzDescription = extractAuthzDescription(route.security);
+
+        description = `${route.options.description ?? ''}${authzDescription ?? ''}`;
```

Comment:
If I understand this correctly, we're appending the authz description with 0 whitespace after the route-provided description. Can we do better than that? Does the description support line breaks/new lines? Having this render on its own line would make it easier to visually parse. If we can't do it on its own line, can we at least consider adding a space between the original description and our authz addition?

![image](https://github.com/user-attachments/assets/e0231fa6-f161-4fd2-b282-4fa5a7c57013)